scripts/docker-build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# This script builds a development TimescaleDB image from the
+# currently checked out source on the host.
+#
+SCRIPT_DIR=$(dirname $0)
+BASE_DIR=${PWD}/${SCRIPT_DIR}/..
+PG_IMAGE_TAG=9.6.3-alpine
+BUILD_CONTAINER_NAME=pgbuild
+IMAGE_NAME=$USER/timescaledb
```

Comment:
Should this image name just be the default. i.e. IMAGE_NAME=${IMAGE_NAME:-$USER/timescaledb}